src/main/java/ca/on/oicr/gsi/dimsum/controller/rest/RunListRestController.java

```diff
@@ -0,0 +1,34 @@
+package ca.on.oicr.gsi.dimsum.controller.rest;
+
+import static ca.on.oicr.gsi.dimsum.controller.mvc.MvcUtils.*;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ca.on.oicr.gsi.dimsum.controller.rest.request.DataQuery;
+import ca.on.oicr.gsi.dimsum.data.Run;
+import ca.on.oicr.gsi.dimsum.service.filtering.RunFilter;
+import ca.on.oicr.gsi.dimsum.service.RunListManager;
+import ca.on.oicr.gsi.dimsum.service.filtering.RunSort;
+import ca.on.oicr.gsi.dimsum.service.filtering.TableData;
+
+@RestController
+@RequestMapping("/rest/runs")
+public class RunListRestController {
+
+  @Autowired
+  private RunListManager runListManager;
+
+  @PostMapping
+  public TableData<Run> query(@RequestBody DataQuery query) {
+    validateDataQuery(query);
+    RunSort sort = parseSort(query, RunSort::getByLabel);
+    boolean descending = parseDescending(query);
+    RunFilter baseFilter = parseBaseRunFilter(query);
```

Comment:
The run list shouldn't have a base filter, as all runs should always be available on it. The base filter is a non-removable filter that's applied to a page, such as filtering by the selected project for the project details page